### PR TITLE
Update decription

### LINKED ISSUE #93: CRAN Comments
### Background Information

Missed to update the acronyms in the DESCRIPTION file.

### Definition of Done

Update Acronyms

### PR DESCRIPTION
**Pull request by rammprasad**
closes #93

### CODE REVIEW

File: DESCRIPTION [hunk 1]
```diff
@@ -3,7 +3,8 @@ Type: Package
 Title: SDTM Data Transformation Engine
```
Comment (edgar-manukyan) [inline DESCRIPTION:3]: Do we need to open the abbreviation here too?

File: DESCRIPTION [hunk 2]
```diff
@@ -3,7 +3,8 @@ Type: Package
...
 Version: 0.1.0
 Authors@R: c(
-    person("Rammprasad", "Ganapathy", role = c("aut", "cre"),
+    person("Rammprasad", "Ganapathy", email = "ganapathy.rammprasad@gene.com",
+    role = c("aut", "cre"),
```
Comment (edgar-manukyan) [inline DESCRIPTION:7]: Was this one one the comments? It seems pipeline is failing and I wonder whether this is causing it.

File: DESCRIPTION [hunk 3]
```diff
@@ -24,13 +25,16 @@ Authors@R: c(
...
-Description: An EDC-agnostic and data standard-agnostic SDTM data transformation engine 
-    that provides a framework for modular programming of SDTM in R. Powered by 
-    metadata and standard specification sdtm.oak can potentially automate the 
-    conversion of raw clinical data to SDTM through standardized mapping 
-    algorithms. SDTM is one of the required standards for data submission to 
-    the FDA (U.S.) and PMDA (Japan). SDTM standards are implemented following 
-    the SDTM Implementation Guide as defined by 
+Description: An Electronic Data Capture systems (EDC) and Data Standard agnostic 
+    solution that enables the pharmaceutical programming community to develop 
+     Clinical Data Interchange Standards Consortium (CDISC)
```
Comment (edgar-manukyan) [inline DESCRIPTION:30]:
```suggestion
    Clinical Data Interchange Standards Consortium (CDISC)
```

There was an extra space.

**Review by edgar-manukyan — APPROVED** [review]

File: DESCRIPTION [hunk 4]
```diff
@@ -24,13 +25,16 @@ Authors@R: c(
...
     )
 Maintainer: Rammprasad Ganapathy <ganapathy.rammprasad@gene.com>
-Description: An EDC-agnostic and data standard-agnostic SDTM data transformation engine 
-    that provides a framework for modular programming of SDTM in R. Powered by 
-    metadata and standard specification sdtm.oak can potentially automate the 
-    conversion of raw clinical data to SDTM through standardized mapping 
-    algorithms. SDTM is one of the required standards for data submission to 
-    the FDA (U.S.) and PMDA (Japan). SDTM standards are implemented following 
-    the SDTM Implementation Guide as defined by 
+Description: An Electronic Data Capture systems (EDC) and Data Standard agnostic
```
Comment (ShiyuC) [inline DESCRIPTION:28]:
```suggestion
Description: An Electronic Data Capture system (EDC) and Data Standard agnostic 
```

File: DESCRIPTION [hunk 5]
```diff
@@ -24,13 +25,16 @@ Authors@R: c(
...
-    metadata and standard specification sdtm.oak can potentially automate the 
-    conversion of raw clinical data to SDTM through standardized mapping 
-    algorithms. SDTM is one of the required standards for data submission to 
-    the FDA (U.S.) and PMDA (Japan). SDTM standards are implemented following 
-    the SDTM Implementation Guide as defined by 
+Description: An Electronic Data Capture systems (EDC) and Data Standard agnostic 
+    solution that enables the pharmaceutical programming community to develop 
+     Clinical Data Interchange Standards Consortium (CDISC)
+    Study Data Tabulation Model (SDTM) datasets in R. The reusable algorithms 
+    concept in 'sdtm.oak' provides a framework for modular programming and  can
```
Comment (ShiyuC) [inline DESCRIPTION:32]:
```suggestion
    concept in 'sdtm.oak' provides a framework for modular programming and can 
```

File: DESCRIPTION (hunk 2 above)
Comment (edgar-manukyan) [reply DESCRIPTION:7]: Please see https://github.com/pharmaverse/sdtm.oak/actions/runs/10605052955/job/29393055019?pr=94

**Review by ShiyuC — COMMENTED** [review]
@rammprasad thanks for updating the description file. I have added 2 minor comments.

**Review by ShiyuC — APPROVED** [review]

File: DESCRIPTION (hunk 1 above)
Comment (rammprasad) [reply DESCRIPTION:3]: I think we might be ok. admiral has ADaM in the title.

**Pull request merged**